pillar.example

```diff
@@ -357,3 +357,7 @@ nginx:
         - alt_nginx.service
       nginx_snippet_file_managed:
         - alt_server.conf
+
+  # Configure formula pillar namespace
+  pillar:
```

Comment:
This `pillar:namespace` parameter and use seems to be something new accross all community formulas, but it may be a solution to a problem @myii @sticky-note are discussing.